.github/workflows/dvc-studio.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-region: us-east-2
-          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-to-assume: ${{ vars.AWS_SANDBOX_ROLE }}
```

Comment:
@0x2b3bfa0 what is the main difference / motivation (just for my education and me adjusting my repos)

Reply:
The difference between `vars` and `secrets` is that the former are meant to store non-sensitive information, without having to hardcode it anywhere; you can see and edit `vars` without any restriction.

As part of the quarterly Access Control Review, we're trying to get rid of as many secrets as possible of the 275 (!) we currently have, to differentiate clearly what's sensitive and what has to be rotated. That's the motivation.

See also [this GitHub blog post](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows), https://github.com/iterative/itops/issues/2422 → https://github.com/iterative/itops/issues/2498 → **https://github.com/iterative/itops/issues/2503**, and [this Slack thread](https://iterativeai.slack.com/archives/C01900GSB4J/p1673363741759559) with some discussion.

Reply:
Feel free to ask for more details on this thread or anywhere on Slack. ™️